src/commands/sdk/execute.ts

```diff
@@ -69,33 +71,70 @@ Other
 
   static readStdin = true;
 
+  async toJavascript(filename: string): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      exec(
```

Comment:
It might be better to use the programmatic api in order to avoid relying on `exec` and to save up on temp file reading / writing.
https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API#a-simple-transform-function